frontend/src/resetUsername.js

```diff
@@ -1,37 +1,117 @@
-import {Logo, Textbox, FormButton} from "./resetPswd.js";
+import {useState, useEffect} from 'react';
 import FormCSS from "./form.module.css";
+import axios from 'axios';
+
+
 
 export default function ResetUsernameForm() {
+    const [email, setEmail] = useState('');
+    const [username, setUsername] = useState('');
+    const [newUsername, setNewUsername] = useState('');
+    const [message, setMessage] = useState('');
+
+    const checkUsername = async () => {
+        try {
+            const response = await axios.get(`http://localhost:8080/users/${newUsername}`)
```

Comment:
the url you are passing in does not relate to any api endpoint you have defined in the username routes, you'll have to change the url to match your endpoint